src/main/java/de/teamlapen/vampirism/proxy/IProxy.java

```diff
@@ -24,4 +27,7 @@ default float getRenderPartialTick() {
     boolean isPlayerThePlayer(EntityPlayer player);
 
     void renderScreenFullColor(int ticksOn, int ticksOff, int color);
+
+    @Nullable
+    Entity getMouseOverEntity(EntityPlayer player);
```

Comment:
Any reason for the player argument? It only works on client side anyway and the argument is not even checked

Reply:
no reason, I probably added it when I made the function signature and forgot to remove it when I didn't end up using it.

Reply:
ok